src/PJ_lccm.c

```diff
@@ -0,0 +1,149 @@
+/***********************************************************************
+
+    This projection is very similar to normal Lambert Conformal Conic
+    2SP (two standard parallels), but with additional scaling factor
+    on the ellipsoid (given as `scale` parameter).
+
+    It is implemented as per EPSG guidance note 7-2 (version 54,
+    August 2018, page 25).
+
+    It is used in a few systems in the EPSG database which justifies
+    adding this otherwise non-standard projection.
+
+ *****************************************************************************/
+
+#define PJ_LIB__
+#include <errno.h>
+#include "proj.h"
+#include "projects.h"
+#include "proj_math.h"
+
+PROJ_HEAD(lccm, "Lambert Conformal Conic (2SP Michigan)")
+    "\n\tConic, Sph&Ell\n\tlat_1= and lat_2= or lat_0";
```

Comment:
Perhaps scale should be mentioned too

Reply:
Ok, should I just add it to the end? i.e. "lat_1= and lat_2= or lat_0, scale="?